### PR TITLE
Add "created" field to token pools, add some extra E2E checks

### PR DESCRIPTION
**Pull request by awrichar**

### CODE REVIEW

File: internal/database/sqlcommon/tokenpool_sql.go [hunk 1]
```diff
@@ -96,6 +97,7 @@ func (s *SQLCommon) UpsertTokenPool(ctx context.Context, pool *fftypes.TokenPool
 			return err
 		}
 	} else {
+		pool.Created = fftypes.Now()
```
Comment (awrichar) [inline internal/database/sqlcommon/tokenpool_sql.go:100]: Where is the appropriate layer to set `created` timestamps? It seems to vary across different objects, but here in the Upsert method felt most logical/efficient to me.

File: docs/swagger/swagger.yaml [hunk 2]
```diff
@@ -4223,6 +4223,11 @@ paths:
         schema:
           default: 120s
           type: string
+      - description: 'Data filter field. Prefixes supported: > >= < <= @ ^ ! !@ !^'
```
Comment (nguyer) [inline docs/swagger/swagger.yaml:4226]: Is there somewhere we can indicate in the docs/swagger that this is a date/time? I had to read further into the source code to see where it gets converted to tell what type it actually was.

**Review by nguyer — APPROVED** [review]
One minor suggestion, otherwise looks good

File: internal/database/sqlcommon/tokenpool_sql.go (hunk 1 above)
Comment (peterbroadhurst) [reply internal/database/sqlcommon/tokenpool_sql.go:100]: Looks appropriate to me 👍

File: docs/swagger/swagger.yaml (hunk 2 above)
Comment (awrichar) [reply docs/swagger/swagger.yaml:4226]: Probably not. The Swagger spec still needs a lot of fleshing out in general... I've noted before that the Swagger generator could almost be a separate project in itself. But I filed https://github.com/hyperledger/firefly/issues/214 for this.

**Pull request merged**